### ISSUE TITLE
Latex not generating properly

### ISSUE THREAD

**Issue by Marti2203** [rostam/GraphTea#57]
Hello,
Recently I tried to generate a latex drawing for a graph and I cannot get the same look as the one I generate in the program. Is there something specific I should do to make it work?

**Comment by azinazadi** [comment]
the latex drawing is a simplified version. it is very hard to make exactly the same look...

**Comment by Marti2203** [comment]
Yeah, I suspect that but still - Assume you have a graph with vertices and edges, I understand that the nodes are hard to make the same but atleast the edges to be visualizable would be good.

**Comment by azinazadi** [comment]
would you please send a photo of what you are getting from the program?
Then I can understand better what you mean...

**Comment by Marti2203** [comment]
In a couple(7-8) of hours I will send a repro.

**Comment by Marti2203** [comment]
![image](https://user-images.githubusercontent.com/7456946/75062335-e89a7480-54e2-11ea-9ab4-f2219eea7312.png)
This is just made. @azinazadi 
[Test.pdf](https://github.com/rostam/GraphTea/files/4237569/Test.pdf)

**Comment by azinazadi** [comment]
thanks. it used to draw also the edges. would you please send also the
generated latex file?

**Comment by Marti2203** [comment]
Compressed as Github does not like .tex files :disappointed: 
[Test.zip](https://github.com/rostam/GraphTea/files/4237977/Test.zip)

**Comment by azinazadi** [comment]
thank you. there is a latex command in the file for the edges. I wonder why it doesnt show up in the pdf. I guess it has to do with the latex compiler, version, etc... @rostam any ideas?

**Comment by Marti2203** [comment]
Maybe i do not have a specific package? I am with the latest latex packages - this has been tested both on Windows 10 machine and currently from an OpenSUSE Tumbleweed.

**Comment by azinazadi** [comment]
Could you check your compile log if it gives any errors or warnings?

Am Dienstag, 25. Februar 2020 schrieb Martin Mirchev <
notifications@github.com>:

**Comment by Marti2203** [comment]
LaTeX Warning: Unused global option(s):
[bezier].
(./Test.aux)
\openout1 = `Test.aux'.
LaTeX Font Info: Checking defaults for OML/cmm/m/it on input line 25.
LaTeX Font Info: ... okay on input line 25.
LaTeX Font Info: Checking defaults for T1/cmr/m/n on input line 25.
LaTeX Font Info: ... okay on input line 25.
LaTeX Font Info: Checking defaults for OT1/cmr/m/n on input line 25.
LaTeX Font Info: ... okay on input line 25.
LaTeX Font Info: Checking defaults for OMS/cmsy/m/n on input line 25.
LaTeX Font Info: ... okay on input line 25.
LaTeX Font Info: Checking defaults for OMX/cmex/m/n on input line 25.
LaTeX Font Info: ... okay on input line 25.
LaTeX Font Info: Checking defaults for U/cmr/m/n on input line 25.
LaTeX Font Info: ... okay on input line 25.
[1
Non-PDF special ignored!
<special> em:linewidth 0.4pt
{/usr/share/texmf/fonts/map/pdftex/updmap/pdftex.map}
Non-PDF special ignored!
<special> em:moveto
Non-PDF special ignored!
<special> em:lineto
Non-PDF special ignored!
<special> em:moveto
Non-PDF special ignored!
<special> em:lineto
Non-PDF special ignored!
<special> em:moveto
Non-PDF special ignored!
<special> em:lineto
] (./Test.aux) )
Here is how much of TeX's memory you used:
273 strings out of 493654
2956 string characters out of 6154966
57846 words of memory out of 5000000
4103 multiletter control sequences out of 15000+600000
3938 words of font info for 15 fonts, out of 8000000 for 9000
475 hyphenation exceptions out of 8191
22i,4n,19p,128b,153s stack positions out of 5000i,500n,10000p,200000b,80000s
</usr/share/texmf/fonts/type1/public/amsfonts/cm/cmr12.pfb></us
r/share/texmf/fonts/type1/public/amsfonts/latxfont/lcircle1.pfb>
Output written on Test.pdf (1 page, 10335 bytes).
PDF statistics:
16 PDF objects out of 1000 (max. 8388607)
10 compressed objects within 1 object stream
0 named destinations out of 1000 (max. 500000)
1 words of extra memory for PDF output out of 10000 (max. 10000000)

**Comment by Marti2203** [comment]
I am using TexMaker, maybe It could be important

**Comment by Marti2203** [comment]
I looked around how GraphTea generates Latex Files and I am interesting in making a new LatexGenerator which uses tikz as a generating backend. @azinazadi @rostam  If it is okay with you guys? If you have any specific requirements, mention them here and I will start working on a generator.

**Comment by azinazadi** [comment]
Sure I think it would be great and very welcome. It would be cool if you
can send a pool request on that, and just make a new format? if you need
help on the way we would be very happy to provide support, thank you.

**Comment by Marti2203** [comment]
I have an important question  - why is the codebase with so many raw types? And I see that there are a couple of dependencies(and official java packages which are deprecated) which I did not know about before I cloned the package.

**Comment by azinazadi** [comment]
Because we developed the base more than 10 years ago and since then we only
added extensions to it... I know that it also doesnt build with newer
versions of Java , needs to add deprecated XML packages... maybe if you
fixed the build, you could also send a pull request on that...

**Comment by Marti2203** [comment]
Nice! 
1) Okay - I will do a major cleanup of the code with a lot of micro commits you can check.
2) I want you guys to know that the program is amazing and I have had a wonderful experience with it and with pleasure fixup the rough edges and build system.

**Comment by Marti2203** [comment]
3. Is the shell still used? @azinazadi

**Comment by rostam** [comment]
@Marti2203 @azinazadi Sorry for late reply. 

@Marti2203 I accepted a new merge request for the support of Java > 8.0. So please update the last commit in your code before sending a merge request for the latex. And you are right, I tested a simple graph and the generated latex file did not work for me. 

1. Please send several pull requests with small changes rather than a big one. 
2. Thanks :) We will appreciate any fixes from you.
3. Yes, we still use shell but I know that the library is very old.
